docs/integrating-into-custom-elements.md

```diff
@@ -0,0 +1,61 @@
+# Integrating MDLv2 into custom elements
```

Comment:
Got it. After discussing offline, we're going to remove this doc and simply touch briefly on custom elements within the contexts of the simple approach of integrating our components into frameworks.